components/CustomText.tsx

```
@@ -0,0 +1,17 @@
+import {Text, TextProps} from 'react-native-paper';
+
+type CustomTextProps = TextProps<{}> & {
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)